vmb/matrimony/models/profiles.py

```diff
@@ -202,6 +207,8 @@ class MatrimonyProfile(BaseModel):
     weight = models.DecimalField(
         max_digits=5, decimal_places=2, help_text="Weight in kgs", null=True,
     )
+    color_of_eyes = models.CharField(max_length=20, null=True, blank=True,)
```

Comment:
This should also become attributes in profile and should be a choice type field.